src/test/java/org/infrastructurebuilder/audit/excel/TestExcelGenerator.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright © 2019 admin (admin@infrastructurebuilder.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.infrastructurebuilder.audit.excel;
+
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.infrastructurebuilder.util.config.TestingPathSupplier;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.UUID;
+
+public class TestExcelGenerator {
+  private final static Logger log = LoggerFactory.getLogger(ExcelGenerator.class);
```

Comment:
`getLogger` should always use the current class (in this case, `TestExcelGenerator.class`)